build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -76,6 +83,7 @@
         <BinariesToRemove Remove="*" />
         <BinariesToRemove Include="csc" />
         <BinariesToRemove Include="vbc" />
+        <BinariesToRemove Include="MSBuild" />
```

Comment:
Is it weird that this Item is called "BinariesToRemove", but I don't see them being removed?